subworkflows/local/prepare_genome.nf

```diff
@@ -184,7 +184,7 @@ workflow PREPARE_GENOME {
     if (prepare_tool_index == 'star') {
         if (params.star_index) {
             if (params.star_index.endsWith('.tar.gz')) {
-                ch_star_index = UNTAR_STAR_INDEX ( params.star_index ).untar
+                ch_star_index = UNTAR_STAR_INDEX ( [], params.star_index ).untar.map{it -> it[1]}
```

Comment:
```suggestion
                ch_star_index = UNTAR_STAR_INDEX ( [], params.star_index ).untar.map{ it[1] }
```

Reply:
Addressed in [33d3bdd](https://github.com/nf-core/chipseq/pull/271/commits/33d3bdda056cc66bd6c9070bc0abf670e9c6c229)